account_payment_order/wizard/account_payment_line_create.py

```diff
@@ -77,8 +76,8 @@ def _prepare_move_line_domain(self):
             domain += [("partner_id", "in", self.partner_ids.ids)]
         if self.target_move == "posted":
             domain += [("move_id.state", "=", "posted")]
-        if not self.allow_blocked:
-            domain += [("blocked", "!=", True)]
+        if not self.order_id.allow_blocked:
```

Comment:
Shouldn't this be `if not self.order_id.payment_mode_id.allow_blocked:`?

Reply:
You're right. Fixed.